Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1224,6 +1224,69 @@ def get_github_actions_usage():
     return_outputs(readable_output=human_readable, outputs=ec, raw_response=usage_result)
 
 
+def list_check_runs(owner_name, repository_name, run_id, commit_id):
+    url_suffix = None
+
+    if run_id:
+        url_suffix = f"/repos/{owner_name}/{repository_name}/check-runs/{run_id}"
+    elif commit_id:
+        url_suffix = f"/repos/{owner_name}/{repository_name}/commits/{commit_id}/check-runs"
+    else:
+        raise DemistoException("You have to specify either the check run id of the head commit reference")
+
+    check_runs = http_request(method="GET", url_suffix=url_suffix)
+
+    return [r for r in check_runs.get('check_runs')]
+
+
+def get_github_get_check_run():
+    """ List github check runs.
+
+    """
+    command_args = demisto.args()
+    owner_name = command_args.get('owner', '')
+    repository_name = command_args.get('repository', '')
+    run_id = command_args.get('run_id', '')
+    commit_id = command_args.get('commit_id', '')
+
+    check_run_result = []
+
+    check_runs = list_check_runs(owner_name=owner_name, repository_name=repository_name, run_id=run_id, commit_id=commit_id)
+
+    for check_run in check_runs:
+        check_run_id = check_run.get('id', '')
+        check_external_id = check_run.get('external_id', '')
+        check_run_name = check_run.get('name', '')
+        check_run_app_name = check_run['app'].get('name', '')
+        check_run_pr = check_run.get('pull_requests')
+        check_run_status = check_run.get('status', '')
+        check_run_conclusion = check_run.get('conclusion', '')
+        check_run_started_at = check_run.get('started_at', '')
+        check_run_completed_at = check_run.get('completed_at', '')
+        check_run_output = check_run.get('output', '')
+
+        check_run_result.append({
+            'CheckRunID': check_run_id,
+            'CheckExternalID': check_external_id,
+            'CheckRunName': check_run_name,
+            'CheckRunAppName': check_run_app_name,
+            'CheckRunPR': check_run_pr,
+            'CheckRunStatus': check_run_status,
+            'CheckRunConclusion': check_run_conclusion,
+            'CheckRunStartedAt': check_run_started_at,
+            'CheckRunCompletedAt': check_run_completed_at,
+            'CheckRunOutPut': check_run_output
+        })
+
+    ec = {
+        'GitHub.CheckRuns(val.ID == obj.CheckRunID)': check_run_result
+    }
+    human_readable = tableToMarkdown('Check Run Details', check_run_result,
+                                     headerTransform=string_to_table_header)
+
+    return_outputs(readable_output=human_readable, outputs=ec, raw_response=check_run_result)
```

Comment:
Please consider using the CommandResults instead of return_outputs

Reply:
> Thanks for your contribution.
> 
>     * Please add release notes. run the `demisto-sdk update-release-notes -i Packs/GitHub -u (major|minor|revision)` to generate release notes according to the new standard. You can refer to the documentation found here: https://xsoar.pan.dev/docs/integrations/changelog for more information.
> 
>     * Let's set up a short demo to see how the fetch works
>       so can you please share your availability?

@moishce I have pushed a new commit with the release notes, I am available anytime Monday first half GST

Reply:
> Please consider using the CommandResults instead of return_outputs

@moishce this is just an additional command that I have added, all existing command functions are using return_outputs , so I thought to follow the pattern,  I don't have time to update all existing command functions to use CommandResults instead of return_outputs

Reply:
@ayman-m Of course, I did not mean to update all existing commands.
Only in the additional command that you have added.

Reply:
@moishce sure, I updated the command to use CommandResults, let have a call when you have a chance this week to demo this update of the integration

Reply:
@moishce  did you have a chance to review  my last commit

Reply:
@moishce a new commit is pushed to reflect what we discussed today